src/main/java/net/dries007/tfc/util/events/StartFireEvent.java

```diff
@@ -32,12 +49,32 @@
  */
 public final class StartFireEvent extends Event implements ICancellableEvent
 {
+    // Replicate vanilla fire starting sounds that do not fire
+    public static boolean startFireWithSound(Level level, BlockPos pos, BlockState state, Direction direction, @Nullable Player player, ItemStack stack)
```

Comment:
Feels a bit hacky to me, but I could not figure out why the vanilla sounds were not playing